src/website/app/Router.js

```diff
@@ -121,7 +131,8 @@ export default function Router({ demoRoutes }: Props) {
         path="/components/:componentId"
         render={route => {
           const componentId = route.match.params.componentId || 'button';
-          const selectedDemo = demoRoutes[componentId];
+          // $FlowFixMe
+          const selectedDemo = flatDemoRoutes[componentId];
```

Comment:
There is a fairly obvious bug here, though not introduced in this PR.  If a bad route is passed in, e.g. a typo in the componentId param, then selectedDemo will be undefined and line 137 will throw an error.  If you will not fix this now, can you please file an issue for followup?

Reply:
I'll file a follow-up to add a proper 404 and include these details.